contracts/staking/StakingTemplate.sol

```diff
@@ -443,6 +443,7 @@ contract StakingTemplate is Ownable {
         // the right amount that delegator can award
         if (currentBlock > lastRewardBlock) {
             uint256 _shareAcc = openedPools[pid].shareAcc;
+            if (_shareAcc == 0 || openedPools[pid].totalStakedAmount == 0) return 0;
```

Comment:
Here i think ```lastRewardBlock != 0``` already guarantee we did have deposit, did you have the issue during testing.

Reply:
Think about this: there has 3pool of the community,add a depositor did deposite to the second pool, so the lastRewardBlock !== 0 since then. But when we need to get the first pools pending reward, the totalStakedAmount ==0, So there will be 0 div issue.

Reply:
I see, you are right! Merge it.